app/views/transactions/_transactions.html.slim

```diff
@@ -1,4 +1,6 @@
 .row
   = render 'transactions/form', transaction: Transaction.new
+  = render 'transactions/transfer_form', transaction: Transaction.new
```

Comment:
@Gataniel why you use Transaction instance here? Looks like this form for creating Transfer?